.circleci/config.yml

```diff
@@ -3,50 +3,17 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/python:3.7.0-stretch-browsers
+    - image: registry.hub.docker.com/mdboom/pyodide:0.1.1
```

Comment:
just `mdboom/pyodide:0.1.1` should be enough unless you prefer to be explicit about the origin?

Reply:
I don't know why it doesn't work, but it doesn't.

```
Unable to find image 'mdboom/pyodide:latest' locally
Trying to pull repository docker.io/mdboom/pyodide ... 
Trying to pull repository registry.fedoraproject.org/mdboom/pyodide ... 
Trying to pull repository quay.io/mdboom/pyodide ... 
Trying to pull repository registry.access.redhat.com/mdboom/pyodide ... 
Trying to pull repository registry.centos.org/mdboom/pyodide ... 
Trying to pull repository docker.io/mdboom/pyodide ... 
/usr/bin/docker-current: manifest for docker.io/mdboom/pyodide:latest not found.
```

Reply:
Yes, but here you are using that `latest` tag which doesn't exist.

```
docker run -it --rm mdboom/pyodide:0.1.1 /bin/bash
```
works for me.